distributions/demo-google-deployment/transfer/build.gradle

```diff
@@ -26,29 +45,69 @@ repositories {
 
 dependencies {
     compile project(':portability-transfer')
-
-    compile project(':extensions:data-transfer:portability-data-transfer-flickr')
+    compile project(':extensions:config:portability-config-yaml')
+    // TODO: depend on these based on list in flag values.
     compile project(':extensions:data-transfer:portability-data-transfer-google')
+    compile project(':extensions:data-transfer:portability-data-transfer-instagram')
     compile project(':extensions:data-transfer:portability-data-transfer-microsoft')
-    compile project(':extensions:cloud:portability-cloud-google')
-    compile project(':extensions:config:portability-config-yaml')
+    compile project(':extensions:data-transfer:portability-data-transfer-flickr')
+    compile project(':extensions:data-transfer:portability-data-transfer-rememberthemilk')
 }
 
 addCloudExtensionDependency(project)
 
 mainClassName = 'org.dataportabilityproject.transfer.WorkerMain'
 
-copyResources()
-
 shadowJar {
     mergeServiceFiles()
     exclude '**/pom.properties'
     exclude '**/pom.xml'
-}
-
-def copyResources() {
     copy {
         from '../resources/'
         into 'src/main/resources'
     }
 }
+
+// Note: This service acct creds file should be used for local testing only. In production, we use
+// GOOGLE_APPLICATION_CREDENTIALS which is stored as a Kubernetes secret.
+task copyTransferServerDeps {
+    copy {
+        from '../bin/service_acct_creds.json'
+        into 'build'
+    }
+}
+
+task createTransferServerDockerfile(type: Dockerfile) {
+    description = 'Builds the Transfer Server Dockerfile'
+    group = 'docker'
+    dependsOn copyTransferServerDeps
+    destFile project.file("${buildDir}/transfer/Dockerfile")
+    from "gcr.io/google-appengine/openjdk:8"
+    exposePort 8082 // Port the TRANSFER server is accessed from
+    copyFile("build/libs/transfer-all.jar", "/app/transfer.jar")
+
+    // Note: Debug port, project ID, and service_acct_creds.json set in the image are for local
```

Comment:
Same here

Reply:
Same - will come back to this in the next PR, will have to refactor the local setup out somehow